tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -186,7 +186,7 @@ void UpdateStats(FertileStatsResource* fertile_stats_resource,
 
     bool is_finished;
     fertile_stats_resource->AddExampleToStatsAndInitialize(
-        data, &target, {example_id}, leaf_id, &is_finished);
+        data, &target, {example_id}, leaf_id, &is_finished, nullptr);
     leaf_lock->unlock();
```

Comment:
please add a comment witht he name of the argument after nullptr /* bla */  (i think it is a map there)